javascript/javascript1/week1/Age-ify.js

```diff
@@ -0,0 +1,4 @@
+const yearOfBirth = 1999;
+let yearFuture = 2045;
+let age = yearFuture - yearOfBirth;
```

Comment:
These variables are never reassigned. Use `const` instead

Reply:
ready to merge now?